### PR TITLE
Announce changes

### PR DESCRIPTION
**Pull request by nileshpatra**
added coc and made the changes

### CODE REVIEW

**Comment by alok760** [comment]
looks good to me.

File: summer_project/coc.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+<a href="https://github.com/ILUGD/ILUGD.github.io" target = "_blank"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/38ef81f8aca64bb9a64448d0d70f1308ef5341ab/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f6461726b626c75655f3132313632312e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_darkblue_121621.png"></a>
+<body class="page-home">
+  <section>
+      <h1>Code of conduct</h1><br>
+      
+      <div align="left">
+        
+        <p><b>General guideline: </b>
+        Contributors and maintainers should participation in our project 
+        and make our community a harassment-free experience for everyone, regardless of age,
```
Comment (hellozee) [inline summer_project/coc.html]: Need to rephrase the line, `Contributors and maintainers should participation in our project` doesn't make any sense

File: summer_project/coc.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            <li><p>Gracefully accepting constructive criticism</p></li>
+            <li><p>Focusing on what is best for the community</p></li>
+        </ul>
+        <h2>Resolving conflicts</h2>
+        <p>Conflict is not bad , healthy debate 
+            and disagreement often yield positive results. However, it is unacceptable
+            to be disrespectful or to engage in behavior that violates the project’s 
+            code of conduct.
+
+            If someone violates the code of conduct, be it mentor or developer , he/she
```
Comment (hellozee) [inline summer_project/coc.html]: How do you ensure if someone has violated the code of conduct or not?

File: summer19/faq.html [hunk 3]
```diff
@@ -5,38 +5,50 @@
...
       <br>
       <div align="left">
+
           <p class="que">Q: What is this programme about?<p>
           <p>A: We at ILUGD have thought of involving people who can dedicate time during this summer to open source projects so that it can help them hone their skills.
         We are still in the planning phase of what projects will be in for this summer , but they will be announced very soon. This is a great opportunity for newbies starting out in open-source to get involved and collaborate with amazing people !</p>
         
         <p class="que">Q: Which communities are monitoring these projects?</p>
-        <p class="">A: India Linux Users Group , Delhi(ILUGD) is taking incharge of it.</p>
+        <p class="">A: India Linux Users Group - Delhi(ILUG-D) is incharge of it. You are expected to read the
```
Comment (hellozee) [inline summer19/faq.html]: Replace this question with, `is there any code of conduct?` and an apt answer

Comment (hellozee) [reply summer19/faq.html]: Also link the COC in the landing page of the Summer Project

File: summer19/faq.html [hunk 4]
```diff
@@ -5,38 +5,50 @@
...
+        <p class="">A: No, this is just an initiative for promoting open-source culture along with doing a solid project. There's no stipend since this is just to hone your skills.</p>
+        <p class="que">Q: What are the starting and ending dates of the program?</p>
         <p class="">A: We will announce it very soon , just stay tuned.</p>
-        <p class="que">Q: How many mentors will be there for eaach project?</p>
+        <p class="que">Q: How many mentors will be there for each project?</p>
         <p class="">A: This will depend on the number of people who apply for the role of mentor . We will let you know when everything is finalized .</p>
+        <p class="que">Q: Will the project be done remotely?</p>
+        <p class="">A: Yes , this is an open source project and will be accomplished remotely. </p>
+        <p class="que">Q: Will anybody who has applied for the role of mentor be allowed to mentor?</p>
+        <p class="">A: No , once the registration for is closed , we will see and sort out people who will finally be mentoring. </p>
```
Comment (hellozee) [inline summer19/faq.html]: There is nothing wrong with the above answer but I feel, the tone is a little bit harsh, replace it with something like, `Once the registrations are closed, mentors will be shortlisted by us according to the projects selected.` and replace the question with something like, `How will be the projects and mentors selected?`

File: summer19/index.html [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    <link rel="stylesheet" type="text/css" href="../assets/css/main.css">
+</head>
+<a href="https://github.com/ILUGD/ILUGD.github.io" target = "_blank"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/38ef81f8aca64bb9a64448d0d70f1308ef5341ab/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f6461726b626c75655f3132313632312e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_darkblue_121621.png"></a>
+<body class="page-home">
+  <section><h1>SUMMER PROJECT 2019 - ILUG-D</h1>
+      <div align="left">
+        <p>
+Hey folks,
+
+We’re planning ILUG-D community project for the summer. If you’re in for some real challenge this holiday season and commit some time to solve a problem with open source solution, the time has come !!
```
Comment (hellozee) [inline summer19/index.html]: Change this to, `We’re planning ILUG-D community project for the summer.` to `We’re planning to host a bunch of community projects for the summer.`

**Comment by hellozee** [comment]
Looks good to me, pushing the changes

**Pull request merged**